<details><summary>Failed to submit graph for 4ea052671e621beadc8d44c80596c39e418cb3c1
</summary>Message:
Failed to run command: 'pbp generate from-feedstock --output /tmp/tmprv2kl5bt/torchvision-feedstock.yaml --custom-metadata-file /tmp/tmprv2kl5bt/custom_metadata.yaml --release-name torchvision-feedstock-15-4ea052671e621beadc8d44c80596c39e418cb3c1 torchvision-feedstock/recipe', with error: 'INFO:root:Found feedstock torchvision-feedstock at /tmp/tmprv2kl5bt/torchvision-feedstock/recipe
INFO:root:Build/Test graph from torchvision-feedstock
Created by sbx_user1051 on 2025-12-10 13:18:13
Command line: /opt/conda/envs/webhook-listener/bin/pbp generate from-feedstock --output /tmp/tmprv2kl5bt/torchvision-feedstock.yaml --custom-metadata-file /tmp/tmprv2kl5bt/custom_metadata.yaml --release-name torchvision-feedstock-15-4ea052671e621beadc8d44c80596c39e418cb3c1 torchvision-feedstock/recipe

INFO:root:Found abs.yaml in /tmp/tmprv2kl5bt/torchvision-feedstock
INFO:root:Creating graph from feedstock at: torchvision-feedstock
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = linux-64.
INFO:pbpcli.utils.expand_variants:Elapsed time for get_package_variants: 0.09 seconds
INFO:pbpcli.utils.expand_variants:Matrix vars: 3 unchanging vars:(397), changing vars (3): ['blas_impl', 'gpu_variant', 'python']
INFO:pbpcli.utils.expand_variants:Elapsed time for preparation for render: 0.00 seconds
/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py:546: UserWarning: The environment variable 'BUILD_VERSION' is being passed through with value '0.22.1'.  If you are splitting build and test phases with --no-test, please ensure that this value is also set similarly at test time.
  warnings.warn(
WARNING: failed to get package records, retrying.  exception was: Unsatisfiable dependencies for platform linux-64: {MatchSpec("libtorch=2.7[build=*cuda128*]")}
Encountered problems while solving:
  - unsupported request
  - unsupported request

Could not solve for environment specs
The following packages are incompatible
├─ libtorch =2.7 *cuda128* does not exist (perhaps a typo or a missing channel);
└─ pytorch =2.7 *cuda128* does not exist (perhaps a typo or a missing channel).
ERROR: Failed to get package records, max retries exceeded.
INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py", line 895, in get_install_actions
    _actions = _install_actions(prefix, index, specs, subdir=subdir)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py", line 1298, in install_actions
    txn = solver.solve_for_transaction(prune=False, ignore_pinned=False)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/core/solve.py", line 156, in solve_for_transaction
    unlink_precs, link_precs = self.solve_for_diff(
                               ~~~~~~~~~~~~~~~~~~~^
        update_modifier,
        ^^^^^^^^^^^^^^^^
    ...<5 lines>...
        should_retry_solve,
        ^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/core/solve.py", line 225, in solve_for_diff
    final_precs = self.solve_final_state(
        update_modifier,
    ...<4 lines>...
        should_retry_solve,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 181, in solve_final_state
    out_state = self._solving_loop(in_state, out_state, index)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/common/io.py", line 85, in decorated
    return f(*args, **kwds)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 308, in _solving_loop
    solved, outcome = self._solve_attempt(in_state, out_state, index, attempt=attempt)
                      ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 365, in _solve_attempt
    new_conflicts = self._maybe_raise_for_problems(outcome, index, out_state, old_conflicts)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 773, in _maybe_raise_for_problems
    self._maybe_raise_for_conda_build(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        {**parsed_problems["conflicts"], **parsed_problems["not_found"]},
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        message=self._prepare_problems_message(unsolvable, index.db, out_state),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 863, in _maybe_raise_for_conda_build
    raise exc
conda_libmamba_solver.conda_build_exceptions.ExplainedDependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("libtorch=2.7[build=*cuda128*]")}
Encountered problems while solving:
  - unsupported request
  - unsupported request

Could not solve for environment specs
The following packages are incompatible
├─ libtorch =2.7 *cuda128* does not exist (perhaps a typo or a missing channel);
└─ pytorch =2.7 *cuda128* does not exist (perhaps a typo or a missing channel).

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py", line 895, in get_install_actions
    _actions = _install_actions(prefix, index, specs, subdir=subdir)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py", line 1298, in install_actions
    txn = solver.solve_for_transaction(prune=False, ignore_pinned=False)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/core/solve.py", line 156, in solve_for_transaction
    unlink_precs, link_precs = self.solve_for_diff(
                               ~~~~~~~~~~~~~~~~~~~^
        update_modifier,
        ^^^^^^^^^^^^^^^^
    ...<5 lines>...
        should_retry_solve,
        ^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/core/solve.py", line 225, in solve_for_diff
    final_precs = self.solve_final_state(
        update_modifier,
    ...<4 lines>...
        should_retry_solve,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 181, in solve_final_state
    out_state = self._solving_loop(in_state, out_state, index)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/common/io.py", line 85, in decorated
    return f(*args, **kwds)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 308, in _solving_loop
    solved, outcome = self._solve_attempt(in_state, out_state, index, attempt=attempt)
                      ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 365, in _solve_attempt
    new_conflicts = self._maybe_raise_for_problems(outcome, index, out_state, old_conflicts)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 773, in _maybe_raise_for_problems
    self._maybe_raise_for_conda_build(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        {**parsed_problems["conflicts"], **parsed_problems["not_found"]},
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        message=self._prepare_problems_message(unsolvable, index.db, out_state),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 863, in _maybe_raise_for_conda_build
    raise exc
conda_libmamba_solver.conda_build_exceptions.ExplainedDependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("libtorch=2.7[build=*cuda128*]")}
Encountered problems while solving:
  - unsupported request
  - unsupported request

Could not solve for environment specs
The following packages are incompatible
├─ libtorch =2.7 *cuda128* does not exist (perhaps a typo or a missing channel);
└─ pytorch =2.7 *cuda128* does not exist (perhaps a typo or a missing channel).

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py", line 895, in get_install_actions
    _actions = _install_actions(prefix, index, specs, subdir=subdir)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py", line 1298, in install_actions
    txn = solver.solve_for_transaction(prune=False, ignore_pinned=False)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/core/solve.py", line 156, in solve_for_transaction
    unlink_precs, link_precs = self.solve_for_diff(
                               ~~~~~~~~~~~~~~~~~~~^
        update_modifier,
        ^^^^^^^^^^^^^^^^
    ...<5 lines>...
        should_retry_solve,
        ^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/core/solve.py", line 225, in solve_for_diff
    final_precs = self.solve_final_state(
        update_modifier,
    ...<4 lines>...
        should_retry_solve,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 181, in solve_final_state
    out_state = self._solving_loop(in_state, out_state, index)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/common/io.py", line 85, in decorated
    return f(*args, **kwds)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 308, in _solving_loop
    solved, outcome = self._solve_attempt(in_state, out_state, index, attempt=attempt)
                      ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 365, in _solve_attempt
    new_conflicts = self._maybe_raise_for_problems(outcome, index, out_state, old_conflicts)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 773, in _maybe_raise_for_problems
    self._maybe_raise_for_conda_build(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        {**parsed_problems["conflicts"], **parsed_problems["not_found"]},
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        message=self._prepare_problems_message(unsolvable, index.db, out_state),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 863, in _maybe_raise_for_conda_build
    raise exc
conda_libmamba_solver.conda_build_exceptions.ExplainedDependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("libtorch=2.7[build=*cuda128*]")}
Encountered problems while solving:
  - unsupported request
  - unsupported request

Could not solve for environment specs
The following packages are incompatible
├─ libtorch =2.7 *cuda128* does not exist (perhaps a typo or a missing channel);
└─ pytorch =2.7 *cuda128* does not exist (perhaps a typo or a missing channel).

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/bin/pbp", line 10, in <module>
    sys.exit(cli())
             ~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1442, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1363, in main
    rv = self.invoke(ctx)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1226, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 794, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 413, in wrapper
    return func(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 624, in from_feedstock
    build_configs, test_configs = expand_build_matrix(feedstock.recipe_dir_path, src_channels)
                                  ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 79, in expand_build_matrix
    _targets, build_configs, test_configs = variants_for_feedstock_and_platform(
                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        recipe_dir_path, platform, src_channels
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/utils/expand_variants.py", line 544, in variants_for_feedstock_and_platform
    metadata_tuples = api.render(
        parsed.recipe,
    ...<2 lines>...
        variants=None,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/api.py", line 60, in render
    metadata_tuples = render_recipe(
        recipe_path,
    ...<4 lines>...
        permit_unsatisfiable_variants=permit_unsatisfiable_variants,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 1078, in render_recipe
    return distribute_variants(
        m,
    ...<3 lines>...
        bypass_env_check=bypass_env_check,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 967, in distribute_variants
    mv.parse_until_resolved(
    ~~~~~~~~~~~~~~~~~~~~~~~^
        allow_no_other_outputs=allow_no_other_outputs,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        bypass_env_check=bypass_env_check,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 1434, in parse_until_resolved
    self.parse_again(
    ~~~~~~~~~~~~~~~~^
        permit_undefined_jinja=False,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        allow_no_other_outputs=allow_no_other_outputs,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        bypass_env_check=bypass_env_check,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 1339, in parse_again
    self._get_contents(
    ~~~~~~~~~~~~~~~~~~^
        permit_undefined_jinja,
        ^^^^^^^^^^^^^^^^^^^^^^^
        allow_no_other_outputs=allow_no_other_outputs,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        bypass_env_check=bypass_env_check,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ),
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 2097, in _get_contents
    rendered = template.render(environment=env)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/jinja2/environment.py", line 1295, in render
    self.environment.handle_exception()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/jinja2/environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/tmp/tmprv2kl5bt/torchvision-feedstock/recipe/meta.yaml", line 80, in top-level template code
    # ffmpeg only supported on linux, and also unsupported on python 3.9: https://github.com/pytorch/vision/blob/v0.20.1/setup.py#L371
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/jinja_context.py", line 311, in pin_compatible
    pins, _, _ = get_env_dependencies(m, "host", m.config.variant)
                 ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 162, in get_env_dependencies
    precs = environ.get_package_records(
        tmpdir,
    ...<11 lines>...
        channel_urls=tuple(m.config.channel_urls),
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py", line 944, in get_install_actions
    precs = get_package_records(
        prefix,
    ...<12 lines>...
        channel_urls=tuple(channel_urls),
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py", line 944, in get_install_actions
    precs = get_package_records(
        prefix,
    ...<12 lines>...
        channel_urls=tuple(channel_urls),
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py", line 944, in get_install_actions
    precs = get_package_records(
        prefix,
    ...<12 lines>...
        channel_urls=tuple(channel_urls),
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py", line 895, in get_install_actions
    _actions = _install_actions(prefix, index, specs, subdir=subdir)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py", line 1298, in install_actions
    txn = solver.solve_for_transaction(prune=False, ignore_pinned=False)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/core/solve.py", line 156, in solve_for_transaction
    unlink_precs, link_precs = self.solve_for_diff(
                               ~~~~~~~~~~~~~~~~~~~^
        update_modifier,
        ^^^^^^^^^^^^^^^^
    ...<5 lines>...
        should_retry_solve,
        ^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/core/solve.py", line 225, in solve_for_diff
    final_precs = self.solve_final_state(
        update_modifier,
    ...<4 lines>...
        should_retry_solve,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 181, in solve_final_state
    out_state = self._solving_loop(in_state, out_state, index)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/common/io.py", line 85, in decorated
    return f(*args, **kwds)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 308, in _solving_loop
    solved, outcome = self._solve_attempt(in_state, out_state, index, attempt=attempt)
                      ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 365, in _solve_attempt
    new_conflicts = self._maybe_raise_for_problems(outcome, index, out_state, old_conflicts)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 773, in _maybe_raise_for_problems
    self._maybe_raise_for_conda_build(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        {**parsed_problems["conflicts"], **parsed_problems["not_found"]},
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        message=self._prepare_problems_message(unsolvable, index.db, out_state),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_libmamba_solver/solver.py", line 863, in _maybe_raise_for_conda_build
    raise exc
conda_libmamba_solver.conda_build_exceptions.ExplainedDependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("libtorch=2.7[build=*cuda128*]")}
Encountered problems while solving:
  - unsupported request
  - unsupported request

Could not solve for environment specs
The following packages are incompatible
├─ libtorch =2.7 *cuda128* does not exist (perhaps a typo or a missing channel);
└─ pytorch =2.7 *cuda128* does not exist (perhaps a typo or a missing channel).
'
aws lambda request id: 4fdd0a27-0e7e-5e2d-9266-6b947b303171
[View in Grafana](https://grafana.anacondaconnect.com/explore?schemaVersion=1&panes=%7B%2286g%22:%7B%22datasource%22:%22loki-central%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7B__aws_cloudwatch_log_group%3D%5C%22%2Faws%2Flambda%2Fpbp-graph-submitter-prod%5C%22%7D%20%7C%20json%20%7C%20requestId%20%3D%20%604fdd0a27-0e7e-5e2d-9266-6b947b303171%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki-central%22%7D,%22editorMode%22:%22builder%22,%22direction%22:%22backward%22%7D%5D,%22range%22:%7B%22from%22:%22now-3h%22,%22to%22:%22now%22%7D,%22panelsState%22:%7B%22logs%22:%7B%22columns%22:%7B%220%22:%22Time%22,%221%22:%22Line%22%7D,%22visualisationType%22:%22table%22,%22labelFieldName%22:%22labels%22%7D%7D%7D%7D&orgId=1)</details>

--- a/abs.yaml
+++ b/abs.yaml
@@ -2,3 +2,6 @@
 # variant, so it's specified for both.
 extra_labels_for_os:
   osx-arm64: [ventura]
+
+channels:
+  - jrobertson

